greynoise/config.schema.yaml

```
@@ -0,0 +1,8 @@
+---
+
+gn_api_key:
+    description: "API Key for authenticating to GreyNoise API"
+    type: "string"
+    secret: true
+    required: true
+    default: "{{ st2kv.system.gn_api_key }}"
```

I would rename the key to something more explicit. If you're an ST2 admin unfamiliar with GreyNoise, and you're listing st2kv keys when you run across one named `gn_api_key`, are you going to know what it's for?

A better name is probably `greynoise_api_key`.

```suggestion
    default: "{{ st2kv.system.greynoise_api_key }}"
```

---

updated.